contracts/external/cw-payroll/src/msg.rs

```diff
@@ -82,6 +86,12 @@ pub struct StreamResponse {
     pub paused: bool,
     pub title: Option<String>,
     pub description: Option<String>,
```

Comment:
```suggestion
    /// Whether the payroll contract is currently paused
    pub paused: bool,
    /// Human readable title for this contract
    pub title: Option<String>,
    /// Human readable description for this payroll contract
    pub description: Option<String>,
```

Perhaps, not 100% necessary, but makes for nice generated docs.